docs/source/cpp/compute.rst

```diff
@@ -1816,25 +1816,25 @@ Structural transforms
   The output type is an Array of items for the ``FIRST``/``LAST`` options
   and an Array of List of items for the ``ALL`` option.
 
-* \(6) Extract a child value based on a sequence of indices passed in
+* \(6) Extract a child value based on a field name or a sequence of indices passed in
```

Comment:
Thinking about this once more, I realize this are the C++ docs, and at that level, the argument is actually a FieldRef, which can be defined by a sequence of indices or string names. The dotted path as how you added in the examples below still needs to be converted to a FieldRef manually (`FieldRef::FromDotPath`).

So maybe we should generalize this to "based on a field reference passed in the options (this field reference can be specified as an index or name. or a sequence thereof)" ?